pkg/lib/serverTime.js

```diff
@@ -321,13 +295,49 @@
                     console.warn("failed to load time servers", error);
                     return result;
                 });
-    };
+    }
```

Comment:
This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-17992-20221205-123635-07fec9b3-fedora-36-devel//Coverage/lcov/github-pr.diff.gcov.html)